juju/juju
# Gofmt static analysis linter

## Description of change

> Why is this change needed?

The following changes introduce the gofmt linter. Most of the code
changes are the consequence of getting the linter to pass. Using the
`make simplify` command does this automatically for you.

## QA steps

> How do we verify that the change works?

This should pass CI bots and local testing either by git hook or `./scripts/gometalinter.bash`

## Documentation changes

> Does it affect current user workflow? CLI? API?

Local developer work flow
 - Git push hook
 - CI Bot
 - `./scripts/verify.bash`

## Bug reference

> Does this change fix a bug? Please add a link to it.

No
